test.js

```diff
@@ -1,29 +1,28 @@
 'use strict'
 const test = require('tape')
-const ignorable = require('ignorable')
+const ignorable = require('./index')
```

Comment:
curious: how come this worked on your machine? same for example.js (but i didn't commit that change)

Reply:
This package use [package-preview](https://github.com/zkochan/package-preview)

Reply:
wasn't working for me 🤔 
i'll have another look when i pick up the PR again